### ISSUE TITLE
MoA

### ISSUE THREAD

**Issue by thuhrw** [qiaolian9/Pruner#3]
文章里写到During the online cost model update of each tuning round, we treat the cross-platform pre-trained cost model as a Siamese network and use its weights to initialize the target model, then fine-tune it through collected online data. Notably, we finally use a momentum update strategy according to the gradients of the target model to adjust the weights of the Siamese network, similar to MoCo.
请问MoA的部分是每轮都会用Siamese network初始化target model吗，如果这样的话是不是不用target model直接用Siamese network自己得到梯度，按照动量更新的策略更新参数就可以了呢？另外动量更新的公式，如果是与MoCo方法相似的话，是否应该是参数动量平均，而不是使用梯度？
如果我有理解错误的地方，请指正，打扰了
